docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
Well then, some more random thoughts:
`Aggregator` (as in [event management](http://www.donnfelker.com/wp-content/uploads/WindowsLiveWriter/UsingPrismsEventAggregator_70CA/EventAggregator_3.png))
`Invoker` (as in Command Pattern)
`Container` (based on @matthewwithanm's suggestion, see [ServiceWorker](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API))

One other thought: could you change it to a noun, something like `CombinedAction` or `ActionStream`, or, I dunno, `MulticastAction`? I mean, it seems like more of an extension/decoration of Actions to me than a separate object that manages them. But I am still finding my way around, could be way off base.